ramls/schemas/profiles/mappingProfile.json

```diff
@@ -0,0 +1,57 @@
+{
+  "$schema":"http://json-schema.org/draft-04/schema#",
+  "description":"Mapping Profile schema",
+  "type":"object",
+  "additionalProperties":false,
+  "properties":{
+    "id":{
+      "description":"Unique identifier",
+      "$ref":"../../raml-util/schemas/uuid.schema"
+    },
+    "name":{
+      "description":"Mapping Profile name",
+      "type":"string"
+    },
+    "description":{
+      "description":"Mapping Profile description",
+      "type":"string"
+    },
+    "recordTypes":{
+      "description":"Mapping Profile records type",
+      "type":"array",
+      "items":{
+        "type":"string",
+        "$ref":"recordTypes.json"
+      }
+    },
+    "mappingProfileRules":{
+      "description":"List of mapping profile rules",
+      "type":"array",
+      "items":{
+        "type":"object",
+        "$ref":"mappingProfileRule.json"
+      }
+    },
+    "userInfo":{
+      "description":"First name, last name and username of the user, who updated the Mapping Profile",
+      "type":"object",
+      "$ref":"userInfo.json"
+    },
+    "outputFormat":{
```

Comment:
We can make this an Enum, as we will be supporting only few types as output
Just have MARC as one of the values for now, we can add others as we go

Reply:
Made as ENUM